internal/github/utils.go

```diff
@@ -0,0 +1,25 @@
+package github
+
+import (
+	"net/http"
+
+	gh "github.com/google/go-github/v33/github"
+)
+
+// parseBoolResponse determines the boolean result from a GitHub API response.
+// Several GitHub API methods return boolean responses indicated by the HTTP
+// status code in the response (true indicated by a 204, false indicated by a
+// 404). This helper function will determine that result and hide the 404
+// error if present. Any other error will be returned through as-is.
+func parseBoolResponse(err error) (bool, error) {
```

Comment:
This name seems not really correct: this method parses error into bool, not the response or bool, so I'd change it into `isErrNotFound(error)`

Reply:
isErrNotFound does not seem correct too. 
I agree, function operates only with error. But at result it determines the condition of the response(true, false or error), not only the existence or type of error.
In other words it resolves response status by error.